pom.xml

```diff
@@ -122,16 +122,35 @@
         <version>3.4.1</version>
       </dependency>
       <dependency>
-        <groupId>junit</groupId>
-        <artifactId>junit</artifactId>
-        <version>4.13.2</version>
+        <groupId>org.junit.jupiter</groupId>
+        <artifactId>junit-jupiter-api</artifactId>
+        <version>5.8.1</version>
+        <scope>test</scope>
+      </dependency>
+      <dependency>
+        <groupId>org.junit.jupiter</groupId>
+        <artifactId>junit-jupiter-engine</artifactId>
+        <version>5.8.1</version>
+        <scope>test</scope>
+      </dependency>
+      <dependency>
+        <groupId>org.junit.vintage</groupId>
+        <artifactId>junit-vintage-engine</artifactId>
```

Comment:
Are there left `JUnit4` tests in project?
If no probably this is not needed.

Reply:
When left out there's a number of tests that fail or error, since `org.codehaus.plexus.PlexusTestCase` extends `junit.framework.TestCase`.

```
java.lang.NoClassDefFoundError: junit/framework/TestCase
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:756)
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
	at java.net.URLClassLoader.defineClass(URLClassLoader.java:468)
	at java.net.URLClassLoader.access$100(URLClassLoader.java:74)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:369)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:363)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:362)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
	at org.apache.maven.plugins.enforcer.MockProject.getBasedir(MockProject.java:351)
	at org.apache.maven.plugin.PluginParameterExpressionEvaluator.<init>(PluginParameterExpressionEvaluator.java:102)
	at org.apache.maven.plugins.enforcer.EnforcerTestUtils.getHelper(EnforcerTestUtils.java:135)
	at org.apache.maven.plugins.enforcer.EnforcerTestUtils.getHelper(EnforcerTestUtils.java:113)
	at org.apache.maven.plugins.enforcer.TestEvaluateBeanshell.testRulePass(TestEvaluateBeanshell.java:69)
```

Reply:
We could minimize the dependency to junit 4 here, or try to remove PlexusTestCase. Although ideally I'd tackle that as a separate MR, given it's likely more involved and could polute the diff here.

Reply:
Done & documented; suggest we create the issue if this is merged separately.